src/features/users/usersSlice.ts

```diff
@@ -74,7 +74,7 @@ type JWTPayload = {
 };
 
 const initialState: UsersState = {
-  token: localStorage.getItem('token') || '',
+  token: api.getCookie('token'),
```

Comment:
Would it be good that the `getCookie` function returns `undefined` or `null` to distinguish between empty string values and unset values?